x-pack/plugins/stack_connectors/common/openai/types.ts

```diff
@@ -18,12 +18,23 @@ import {
   InvokeAIActionResponseSchema,
 } from './schema';
 
-export type Config = TypeOf<typeof ConfigSchema>;
-export type Secrets = TypeOf<typeof SecretsSchema>;
```

Comment:
Why the change?  Now the type no longer matches the schema - `certPath` is not optional when used with `apiProvider: PkiOpenAi`, but this isn't reflected in the type.